front/src/applications/operationalStudies/views/Scenario/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -983,12 +986,22 @@ const handleNodeOperation = async ({
         if (indexNode.dbId) {
           // Update the key if trigram has changed and key is based on it
           let nodeKey = indexNode.path_item_key;
+          let newTrigram: string | null = null;
           if (nodeKey.startsWith('trigram:') && indexNode.trigram !== node.betriebspunktName) {
-            nodeKey = `trigram:${node.betriebspunktName}`;
+            const trigram = node.betriebspunktName;
+            nodeKey = `trigram:${trigram}`;
+            if (trigram.split('/').length < 2) {
```

Comment:
Nit: `trigram.includes('/')` would be a simpler way to write this.

Reply:
You're right, done